moveit_ros/moveit_servo/launch/servo_example.launch.py

```diff
@@ -125,17 +125,6 @@ def generate_launch_description():
                 name="static_tf2_broadcaster",
                 parameters=[{"child_frame_id": "/panda_link0", "frame_id": "/world"}],
             ),
-            ComposableNode(
```

Comment:
Line 155 handles it

Reply:
Could we add a comment explaining why we are launching servo server as a separate node?

Also should we delete the comment block "# Example of launching Servo as a node component" ?

Reply:
I added some better comments  :+1: